src/routes/(main)/auth/register/form-schema.ts

```diff
@@ -0,0 +1,29 @@
+import { type } from 'arktype';
```

Comment:
Form schema in a separate file to allow the import on both the server and client for the arktype validation.